src/qiboml/backends/__init__.py

```diff
@@ -7,6 +7,7 @@
 from qiboml.backends.tensorflow import TensorflowBackend
 
 PLATFORMS = ["tensorflow", "pytorch", "jax"]
+AVAILABLE_PLATFORMS = ["tensorflow"]
```

Comment:
```suggestion
AVAILABLE_PLATFORMS = ["tensorflow"] # temporary: to remove once pytorch and tensorflow are migrated  and jax is fully woring
```

this is only temporary, right? I would write a reminder then